Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml

```
@@ -124,11 +124,12 @@
         Heat: 0.35
         Caustic: 0.5
         Stamina: 0.3 #SS220 stamina resist fix
+        Radiation: 0.85 #ss220 raid suit rework
   - type: ExplosionResistance
     damageCoefficient: 0.35
   - type: ClothingSpeedModifier
-    walkModifier: 0.9
-    sprintModifier: 0.9
+    walkModifier: 1 #ss220 raid suit rework
```

```suggestion
    walkModifier: 0.9 #ss220 raid suit rework
```
Так как walk > sprint. Так не должно быть

---

Ну скорость ХОДЬБЫ вообще почти ничего не значит, ибо зачем на ЯО скорость шага (ходить на шифте)?

---

То, что костюм не понижает скорость ходьбы - просто удобно, никакого разительного дизбаланса в этом нет.

---

Да и есть скаф джаггернаута у яо, у которого sprint 0.65, а walk 0.9

---

@Ady4ik Исправь пиво